functions-and-scope.js

```diff
@@ -14,13 +14,44 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+let teller = 0;
+for (let i = 0; i < grades.length ; i++) {
+    const currentGrade = grades[i];
+    if (currentGrade >= 8){
+        teller = teller +1;
+    }
+}
+console.log(teller);
+
 // ---- Verwachte uitkomst: 6
 
 
 /*  1b: Omschrijven tot een herbruikbare functie   */
 // Schrijf een functie genaamd cumLaude, die een array van cijfers verwacht (zoals grades) en het aantal Cum laude studenten teruggeeft. Gebruik hiervoor jouw antwoord van 1a.
 // Zorg ervoor dat jouw functie ook werkt als we een andere array met eindcijfers willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
+function cumLaude(arrayCijfers) {
+    let teller = 0;
+    for (let i = 0; i < arrayCijfers.length ; i++) {
+        const currentGrade = arrayCijfers[i];
+        if (currentGrade >= 8){
+            teller = teller +1;
+            //console.log(teller)
+        }
+    }
+    return teller;
+}
+const eerste = cumLaude(grades);
+console.log(eerste);
+console.log(cumLaude(grades));
+
+const tweede = cumLaude([6,4,5]);
+console.log(tweede);
+console.log(cumLaude([6,4,5]));
+
+const derde = cumLaude([8,9,4,6,10]);
+console.log(derde);
+console.log(cumLaude([8,9,4,6,10]));
```

Comment:
Hoi Marck, je uitkomst wordt dubbel gelogd 1 van de 2 regels 53 of 54 kun je weglaten. Geldt ook voor regel 45,46 en 49, 50.